app/main.py

```diff
@@ -0,0 +1,36 @@
+from functools import lru_cache
+from fastapi import FastAPI
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+from .routers import metrics
+
+
+class Settings(BaseSettings):
+    stac_url: str = "http://localhost:8080"
+    env: str = "dev"
+
+    model_config = SettingsConfigDict(env_file=".env")
+
+
+@lru_cache
+def get_settings():
+    return Settings()
+
+
+settings = get_settings()
+app = FastAPI(
+    title="BioTableroSearch",
+    description="Get metrics by predefined or custom (polygon) areas.",
+    summary="Metrics for BioTablero Search module",
+    version="0.1.0",
```

Comment:
Just to confirm, is the version 0.0.1 or 0.1.0? The README indicates it's version 0.0.1

Reply:
0.1.0